modules/kubientBidAdapter.js

```diff
@@ -109,31 +114,37 @@ export const spec = {
     return bidResponses;
   },
   getUserSyncs: function (syncOptions, serverResponses, gdprConsent, uspConsent) {
-    const syncs = [];
-    let gdprParams = '';
+    let kubientSync = kubientGetSyncInclude(config);
+
+    if (!syncOptions.pixelEnabled || kubientSync.image === 'exclude') {
+      return [];
+    }
+
+    let values = {};
+
     if (gdprConsent && typeof gdprConsent.consentString === 'string') {
-      gdprParams = `?consent_str=${gdprConsent.consentString}`;
+      values['consent'] = gdprConsent.consentString;
       if (typeof gdprConsent.gdprApplies === 'boolean') {
-        gdprParams = gdprParams + `&gdpr=${Number(gdprConsent.gdprApplies)}`;
+        values['gdpr'] = Number(gdprConsent.gdprApplies);
```

Comment:
I believe gdpr can apply even if there isnt a consent string, see #7775

Reply:
Plz review again. Plz explain why we can't pass CI tests. I've spent a lot of time to check all this. We made our adapter similar to documentation.